### ISSUE TITLE
Update notifications immediately when confirming/forecasting, unconfirming/unforecasting period

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#780]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
I want the notification for confirming or forecasting the week to disappear immediately after i confirm or forecast the week.

### Description
Right now, the notifications for confirming/forecasting weeks gets updated on page reload/navigation.
Given a scenario where i'm confirming/forecasting multiple weeks in succession, and using the notification UI to navigate between them, I want the current week i've just confirmed to disappear, so I don't accidentally click it and navigate to the same week twice.

This is definitely not a major issue, but would improve the snappiness and "real time"-y feel of the app.

#### Confirmed period, pre page load
![image](https://user-images.githubusercontent.com/7300548/103640908-23699780-4f51-11eb-8b53-c11e58161ac2.png)

#### Confirmed period, post page load
![image](https://user-images.githubusercontent.com/7300548/103640925-295f7880-4f51-11eb-9092-a5ff0c963bc1.png)
